src/services/tweets.sevice.ts

```diff
@@ -0,0 +1,63 @@
+import { ObjectId } from "mongodb";
```

Comment:
did not use database inside services.
Only the repository needs to know which database we are working with.